Exiled.API/Features/Player.cs

```diff
@@ -1184,6 +1186,39 @@ public void ShowHint(string message, float duration = 3f)
             HintDisplay.Show(new TextHint(message, parameters, null, duration));
         }
 
+        /// <summary>
+        ///  Disables all status affects on the player.
+        /// </summary>
+        public void DisableAllEffects()
+        {
+            foreach (KeyValuePair<Type, PlayerEffect> effect in ReferenceHub.playerEffectsController.AllEffects)
+            {
+                effect.Value.ServerDisable();
+            }
+        }
+
+        /// <summary>
+        /// Enables a status effect on this player.
+        /// </summary>
+        /// <typeparam name="T">The <see cref="PlayerEffect"/> to enable.</typeparam>
+        /// <param name="duration">The amount of time the effect will be active for.</param>
+        /// <param name="addDurationIfActive">If the effect is already active, setting to true will add this duration onto the effect.</param>
+        public void EnableEffect<T>(float duration = 0f, bool addDurationIfActive = false)
+        {
+            PlayerEffect playerEffect;
+            if (!ReferenceHub.playerEffectsController.AllEffects.TryGetValue(typeof(T), out playerEffect))
+                return;
```

Comment:
That's useless code, `PlayerEffectController::EnableEffect<T>(float, bool)` already does that